iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -915,6 +916,10 @@ private void setCommonJobConf(JobConf jobConf) {
     jobConf.set("tez.mrreader.config.update.properties", "hive.io.file.readcolumn.names,hive.io.file.readcolumn.ids");
   }
 
+  public StorageHandlerTypes getStorageHandlerType() {
```

Comment:
why not simply `getType`?

Reply:
Even it is an instance, the getType mainly used in several languages to identify the instance type, and I think it is already overused. getHandlerType could be also ok or the long name is ok to me.